src/pages/index.js

```diff
@@ -5,38 +5,39 @@ import {
   resetValidation,
   disabledButton,
 } from "../scripts/validation.js";
+import Api from "../utils/Api.js";
 
-const initialCards = [
-  {
-    name: "Val Thorens",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "39df9e1c-6ec2-44d1-a3b6-60d6e9498d03",
+    "Content-Type": "application/json",
   },
-  {
-    name: "Restaurant terrace",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
-  },
-  {
-    name: "An outdoor cafe",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
-  },
-  {
-    name: "A very long bridge, over the forest and through the trees",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
-  },
-  {
-    name: "Tunnel with morning light",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
-  },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-];
+});
 
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((cardItem) => {
+      const cardElement = getCardElement(cardItem);
+      cardsList.prepend(cardElement);
```

Comment:
**Could be improved**

You can make a universal function for adding a card into the section using different methods: `append`, `prepend`, or anything else:

```js

// The function accepts a card object and a method of adding to the section
// The method is initially `prepend`, but you can pass `append` 
function renderCard(item, method = "prepend") {
  
  const cardElement = getCardElement(item);
  // Add the card into the section using the method
  cardsList[ method ](cardElement);
}



```

Every `DOM` element is an **object** with methods, that’s why you can get the methods in 2 ways: `cardsList.prepend(…)` (with a **dot**) or `cardsList["prepend"](…)`. It’s the same.